ArduSub/GCS_Sub.cpp

```diff
@@ -66,7 +72,9 @@ void GCS_Sub::update_vehicle_sensor_status_flags()
         control_sensors_health |= MAV_SYS_STATUS_SENSOR_OPTICAL_FLOW;
     }
 #endif
-
+    if (visual_odom && visual_odom->enabled() && visual_odom->healthy()) {
```

Comment:
I wonder if this also needs to be surrounded by checks of VISUAL_ODOMETRY_ENABLED